Yosemite/Yosemite/Stores/OrderStore.swift

```diff
@@ -108,6 +108,14 @@ private extension OrderStore {
 
             self?.upsertStoredOrdersInBackground(readOnlyOrders: [order]) {
                 onCompletion(order, nil)
+//                self?.ensureProductsAreSynchronized(for: order, onCompletion: { (products, error) in
+//                    // Even if there was an error while getting products, we want to continue.
+//                    if let error = error {
+//                        DDLogError("Error synchronizing Products for OrderID: \(order.orderID). \(error)")
+//                    }
+//
+//                    onCompletion(order, nil)
+//                })
```

Comment:
This code exists in case we decide that coupling the products remote requests to a successful GET Order by ID is the better solution.